cxx/Makefile

```diff
@@ -20,10 +20,10 @@ INCDIR=$(shell pwd)
 INCFLAGS=-I$(INCDIR)
```

Comment:
You can take a look at the shell script (and modifying that instead).

Reply:
Done. I reinstated test_misc in the integration tests since that's currently the only test covering relations for non-Bernoulli distributions. I'll get rid of it when the unit tests are fleshed out.